examples/usecases/transformers-next-item-prediction.ipynb

```diff
@@ -0,0 +1,1234 @@
+{
```

Comment:
<p>can we change this to return dictonary that we know what these numbers are?</p><p> </p><p>Do we have any Conclusion or Next Steps?</p><p> </p><p>E.g. Next steps could be dressipi example?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/853/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>Absolutely, that is a great idea to link to that notebook! I added another section where I am referencing the dressipi example.</p>